CIP-1694/README.md

```diff
@@ -0,0 +1,1142 @@
+DRep delegation will mimic the existing stake delegation mechanisms (via on-chain certificates).
+Similarly, DRep registration will mimic the existing stake registration mechanisms.
+
+Registered DReps are identified by a credential that can be either:
+
+* A verification key (Ed25519)
+* A native or Plutus script
+
+The blake2b-224 hash digest of a serialized DRep credential is called the _DRep ID_.
+
+The following new types of certificates will be added for governance:
+DRep registration certificates, DRep retirement certificates, and
+vote delegation certificates.
+
+##### DRep registration certificates
+
+DRep registration certificates include:
+
+* a DRep ID
+* an anchor
+
+An **anchor** is a pair of:
+
+* a URL to a JSON payload of metadata
+* a hash of the contents of the metadata URL
+
+The structure and format of this metadata is deliberately left open in this CIP.
+The on-chain rules will not check either the URL or the hash.
+Client applications should, however, perform the usual sanity checks when fetching content from the provided URL.
+
+
+##### DRep retirement certificates
+
+DRep retirement certificates include:
+
+* a DRep ID
+* the epoch number after which the DRep will retire
+
+
+##### Vote delegation certificates
+
+Vote delegation certificates include:
+
+* the DRep ID to which the stake should be delegated
+* the stake credential for the delegator
+
+##### Certificate authorization schemes
+
+The authorization scheme (i.e. which signatures are required for registration, retirement or delegation) mimics the existing stake delegation certificate authorization scheme.
+
+<!-- TODO: Provide CBOR specification in the annexe for those new certificates. -->
+
+
+#### New stake distribution for DReps
+
+In addition to the existing per-stake-credential distribution and the
+per-stake-pool distribution, the ledger will now also determine the per-DRep stake
+distribution. This distribution will determine how much stake is backed by each
+`Yes` vote from a DRep.
+
+> **Warning**
+>
+> **Unlike** the distribution that is used for block production, we will always use the most
+> current version of the per-DRep stake distribution as given on the epoch boundary.
+>
+> This means that **for any topic which individual voters care deeply about,
+> they have time to register themselves as a DRep and vote directly**.
+> However, it means that there may be a difference between the stake that is used for block
+> production and the stake that is used for voting in any given epoch.
+
+
+#### Incentives for Ada holders to delegate voting stake
+
+There will be a short [bootstrapping phase](#bootstrapping-phase) during which rewards will be earned
+for stake delegation etc. and may be withdrawn at any time.
+After this phase, although rewards will continue to be earned for block delegation etc., reward accounts will be
+**blocked from withdrawing any rewards** unless their associated stake credential is also delegated to a DRep
```

Comment:
The intention behind this is increasing the legitimacy of a vote action by increasing the participation. 
"legitimacy" should first be defined, and blocking withdrawals doesn't seem to fulfill achieve the goal of legitimacy.
As per the document, it is meant to prevent "9 out of 10 lovelaces" from being a legitimate vote, but "9 out of 10 lovelaces" voting yes while the rest vote abstain is still possible, and just as unlegitimate, the abstain votes didn't add anything to the legitimacy metric.

Reply:
> As per the document, it is meant to prevent "9 out of 10 lovelaces" from being a legitimate vote, but "9 out of 10 lovelaces" voting yes while the rest vote abstain is still possible, and just as unlegitimate, the abstain votes didn't add anything to the legitimacy metric.

I have worried about this as well. I take a less extreme position, though. I think that there is a substantial different between:

* 10 lovelace are registered to any DReps (including the pre-defined ones). 9/10 lovelace votes yes to proposition P.
* 1 billion ADA are  registered to DReps. 999,999,999,999,990 of these lovelace submit something on chain that makes clear their intention to abstain from the governance. 9/10 lovelace votes yes to proposition P.

Of course I still worry about the second scenario, but it is not as bad as the first.

Reply:
- Scenario 1: all ada except 10 lovelaces ignored P, and may not even be aware of it. Passing P would definitely be bad
- Scenario 2: 1 billion ada is aware of P, doesn't care enough to say yes or no, 9/10 turn the vote into yes. Is passing P bad in this case?

If we agree that passing P in scenario 2 is ok, the current solution works. If we agree that P shouldn't pass, there still needs some form of AVST (maybe a dynamic threshold to solve the long-term participation/liveness problem of the previous AVST idea)

Reply:
It's becoming harder and harder for me to remember what I said where, but we (our game theorists, I should say) are investigating adaptive thresholds.

Reply:
I don't understand why some folks want a perception of `legitimacy`. What would be the purpose? 
The irony of a governance system with perhaps very few exceptions (some countries i have named off in the past ) is that only a relative handful actually participate and is an active choice.

Staking ADA is the most basic way to secure the network. Today we see millions of ada still delegated to dead/retired pools. "Passively"(staking is an active choice) securing the network is a struggle. Forcing participation you might see high numbers at the start but over time I would not doubt those numbers would fall off as people move to exchanges. There could very well be a point were a hard codes threshold will never be meet and where SPOs would then be tapped to step up and vote every single time. I am in the camp where SPOs should be voting every single time since they would be the most active but how i am reading the CIP that should not be the case.

Some people just don't want to participate in any form of governance. Adding on term limits and having to redelegate voting power to dReps periodically would only encourage people to stay on exchanges where they would be insulated from on chain politics.

Reply:
In Tezos they have a running adaptive threshold, with minimum 20% and maximum 80%.
The new quorum threshold takes the old one and slightly moves based on the last participation.
`new-quorum = 0.8 × old-quorum + 0.2 × participation`
link: https://wiki.tezosagora.org/learn/governance/tezos-governance-overview
This solves the long term participation problem, as the threshold would also lower to reflect real participation (assuming at least the minimum 20% always participates).

Not saying this is THE solution (tezos has its own problems), but it's always good to look at existing real world examples.